queries.sql

```diff
@@ -7,4 +7,21 @@ SELECT date_of_birth FROM animals WHERE name = 'Augmon' OR name =  'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name NOT LIKE '%Gabumon%';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3 OR weight_kg = 10.4 OR weight_kg = 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3 OR weight_kg = 10.4 OR weight_kg = 17.3;
+BEGIN;
+UPDATE animals SET species = 'unespecified';
+ROLLBACK;
+UPDATE animals SET species = 'Digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'Pokemon' WHERE name NOT LIKE '%mon';
+DELETE FROM animals;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT first_delete;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO first_delete;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals GROUP BY escape_attempts HAVING escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, COUNT(*) FROM animals GROUP BY neutered;
```

Comment:
- [ ] Kindly make sure this query answer this question: **"Who escapes the most, neutered or not neutered animals?"**(i.e this query should give the animal with the most escape_attempts, neutered or not neutered).